config/nrfconnect/chip-module/Kconfig

```diff
@@ -366,4 +366,11 @@ config CHIP_ENABLE_BDX_LOG_TRANSFER
 	help
 	  Enables the BDX protocol for diagnostics log transfer purposes.
 
+config CHIP_USE_OPENTHREAD_ENDPOINT
+  bool "Use OpenThread TCP/UDP stack directly"
+  depends on OPENTHREAD && !WIFI_NRF70
```

Comment:
How about replacing `WIFI_NRF70` with `WIFI`? I don't think that using specific board/family is the best idea here. Otherwise we will have to update it in the future.